browser/src/control/Control.Menubar.js

```diff
@@ -1491,6 +1492,13 @@ L.Control.Menubar = L.Control.extend({
 							$(aItem).removeClass(constChecked);
 						}
 
+					} else if (id === 'toggleuimode') {
+						if (window.userInterfaceMode === 'notebookbar') {
+							$(aItem).text(_('Use Classic UI'));
+						} else {
+							$(aItem).text(_('Use Notebookbar UI'));
```

Comment:
I think this piece of code is not needed?  NB will use own label defined somewhere else and in classic mode we alwyas need "Use Notebookbar UI" text